descwl_shear_sims/simple_sim.py

```diff
@@ -706,10 +743,13 @@ def _psf_render_func(*, x, y, center_psf):
                     x=x-int(x+0.5),
                     y=y-int(y+0.5),
                 )
-            return psf.drawImage(
+            psf_image = psf.drawImage(
                 nx=self.psf_dim,
                 ny=self.psf_dim,
                 offset=offset,
-                wcs=se_wcs.local(image_pos=image_pos))
+                wcs=psf_wcs,
+                # wcs=se_wcs.local(image_pos=image_pos),
```

Comment:
Why don't we need the `.local` call to make sure we get the right WCS at the PSF location?

Reply:
Does a local wcs know anything about the origin? That info is needed to do the coadd

Reply:
Well galsim when drawing an image calls .local anyways. 

All of this code actually needs to be in the coaddin package. There you should create the coadd wcs you need, draw the psf models at the appropriate location using the sims in modified, and then coadd them.